source/renderer/app/components/wallet/summary/WalletSummary.scss

```diff
@@ -50,26 +50,12 @@
   }
 }
 
-.pendingConfirmation {
-  @extend %smallText;
-  & > .currencySymbolSmallest {
-    display: inline-block;
-    margin-left: 1px;
-    & > svg {
-      height: 10px;
-      width: 9px;
-      & > g > g {
-        stroke: var(
-          --theme-icon-ada-summary-wallet-pending-confirmation-symbol-color
```

Comment:
@tomothespian can you please remove this CSS variable? We are not using it anywhere else anymore...

Reply:
@nikolaglumac oou, nice catch. I will do that ASAP

Reply:
@nikolaglumac Done!

Reply:
Thanks!